### ISSUE TITLE
Line numbers are off by 1 when using JSON formatter

### ISSUE THREAD

**Issue by BobReid** [microsoft/pyright#1005]
**Describe the bug**
Line numbers reported on command line is off by 1 when using the json output formatter. 

**To Reproduce**
run ```pyright --outputjson test.py`` where test.py is the following

```py
def foo():
    pass


foo(1)
```
The reported json results are:
```json
{
    "version": "1.1.65",
    "time": "1599251588036",
    "diagnostics": [
        {
            "file": "/Users/robertreid/resq/resq-fullstack/test.py",
            "severity": "error",
            "message": "Expected 0 positional arguments",
            "range": {
                "start": {
                    "line": 4,
                    "character": 4
                },
                "end": {
                    "line": 4,
                    "character": 5
                }
            }
        }
    ],
    "summary": {
        "filesAnalyzed": 10,
        "errorCount": 1,
        "warningCount": 0,
        "timeInSec": 0.238
    }
}
```

**Expected behavior**
Line number of the issue should be line: 5 and not line 4. 
When running the text output the line numbers are reported correctly. 

```
5:5 - error: Expected 0 positional arguments
```

**VS Code extension or command-line**
command line 
version: pyright 1.1.65

**Comment by erictraut** [comment]
The line and character numbers reported in the JSON output are zero-based. If you want to report them as one-based, simply add one. I've added docs for this [here](https://github.com/microsoft/pyright/blob/master/docs/command-line.md).

**Comment by BobReid** [comment]
Okay that is not intuitive since most source code tools don't treat line counts as 0 based. Documentation around it is welcome thanks!
